meinberlin/apps/cms/migrations/0039_alter_homepage_body.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.16 on 2022-10-20 11:35
+
+from django.db import migrations
+import wagtail.blocks
+import wagtail.fields
+import wagtail.images.blocks
+
+
+def empty_to_valid_json(apps, schema_editor):
+    HomePage = apps.get_model('meinberlin_cms', 'HomePage')
+
+    for homepage in HomePage.objects.all():
+        if len(homepage.body) == 0:
```

Comment:
I'm not sure if this is the best way to check but it seems to work

Reply:
I would say either @fuzzylogic2000 or @Rineee are better people to ask about that?

Reply:
I also don't know if it's the best way, but if it works it's fine! And it makes sense for what it's meant to check. :)

Reply:
@fuzzylogic2000 @goapunk i' happy to merge if you guys are?

Reply:
@goapunk we have a backup db incase it breaks dev right? :p